### PR TITLE
Added a tutorial for the soil equivalent part

### PR DESCRIPTION
**Pull request by EleniSmyrniou**

### CODE REVIEW

File: docs/tutorials.rst [hunk 1]
```diff
@@ -696,6 +696,17 @@ a unique name.
     model.add_soil_layer_by_coordinates(soil2_coordinates, material_soil_2, "soil_layer_2")
     model.add_soil_layer_by_coordinates(embankment_coordinates, material_embankment, "embankment_layer")
 
+Generating the train track
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+STEM provides two options to generate a straight track:
+
+- A straight track with rails, sleepers and rail pads. This track placed on top of the 2D or 3D geometry.
```
Comment (brunozc) [inline docs/tutorials.rst:704]:
```suggestion
1. A straight track with rails, sleepers and rail pads. This track placed on top of the 2D or 3D geometry.
```

File: docs/tutorials.rst [hunk 2]
```diff
@@ -696,6 +696,17 @@ a unique name.
...
+- A straight track with rails, sleepers, rail pads and an extension of the track outside the 2D or 3D geometry.
```
Comment (brunozc) [inline docs/tutorials.rst:705]:
```suggestion
1. A straight track with rails, sleepers, rail pads and an extension of the track outside the 2D or 3D geometry.
```

File: docs/tutorials.rst [hunk 3]
```diff
@@ -718,6 +729,77 @@ are spaced 0.5m from each others which results in a 50m straight track, with par
...
 
+
+**Option 2: Extended straight track with rails, sleepers, rail pads and 1D soil elements.**
+
+When applying a moving train load to the track, we are often interested in the dynamic response of a specific area of
+the track. This requires the generated geometry to be large enough to ensure that the initial train load has not yet
+reached or influenced the area of interest. However, this results in a significantly larger geometry, which, in turn,
+increases computational cost.
+
+To reduce the computational cost, an alternative approach can be used with STEM. The area of interest is still modeled
```
Comment (brunozc) [inline docs/tutorials.rst:740]:
```suggestion
To reduce the computational cost, an alternative approach can be used with STEM. The area of interest is still modelled
```

**Comment by aronnoordam** [comment]
merged in main

**Pull request closed without merging**